app/Http/Requests/ImportMappingRequest.php

```
@@ -12,10 +12,51 @@ public function rules(): array
+
+    public function getName(): string
+    {
+        return (string) $this->input('name');
+    }
+
+    public function getBankName(): ?string
+    {
+        return $this->input('bank_name') ? (string) $this->input('bank_name') : null;
+    }
+
+    public function getColumnMapping(): array
```

PHPStan (2.1.15)

Method App\Http\Requests\ImportMappingRequest::getColumnMapping() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)